### PR TITLE
fix(application request): empty document issue

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Fix empty document issue in application requests detail overlay
Add CSS to document link

## Why

As bpn value was null so condition was not satisfying

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/661

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,8 +14,10 @@
   - UI updates of order status button and back button
 - Service Release Process
   - Updated technicalUserProfile by allowing 'none' for dataspace services
+  - Support optional terms for contract and consent
 - App Release Process
   - Updated technicalUserProfile by allowing 'none'
+  - Support optional terms for contract and consent
```
Comment (manojava-gk) [inline CHANGELOG.md:20]: is this comment relevant?

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,8 +14,10 @@
   - UI updates of order status button and back button
 - Service Release Process
   - Updated technicalUserProfile by allowing 'none' for dataspace services
+  - Support optional terms for contract and consent
```
Comment (manojava-gk) [inline CHANGELOG.md:17]: is this comment relevant?

**Review by manojava-gk — COMMENTED** [review]
small clarification. otherwise looks good to me

File: CHANGELOG.md (hunk 1 above)
Comment (nidhigarg-bmw) [reply CHANGELOG.md]: @manojava-gk Not added by me. May be get added by mistake. Removed them now.

**Review by manojava-gk — APPROVED** [review]
lgtm

**Review by oyo — APPROVED** [review]

**Pull request merged**